setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = alfasim_sdk
+description = ALFAsim API/SDK
+url = https://github.com/esss/alfasim-sdk
+version = 0.1.0
+author = ESSS
+author_email = foss@esss.co
+classifiers =
+    Development Status :: 3 - Pre-Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options.entry_points]
+console_scripts =
+    alfasim-sdk=alfasim_sdk.cli:main
+    alfasim_sdk=alfasim_sdk.cli:main
+
+[options]
+include_package_data = True
+packages =
+    alfasim_sdk
+
+python_requires = >=3.6
+zip_safe = no
+setup_requires =
+    setuptools>=40.0
+    setuptools-scm
```

Comment:
Ahh because we use `setuptools-scm`, we can remove the `MANIFEST.in` altogether: `setuptools-scm` will add all git-tracked files into the package automatically.

Reply:
owww awesome.
I didn't know that `setuptools-scm`  handled this besides the package version.